### DISCUSSION TITLE
Unable to perform install --HEAD

### DISCUSSION THREAD

**Discussion by bohdaq** [Homebrew/discussions#4071] [Writing Formulae/Casks] [answered]
[Gist](https://gist.github.com/bohdaq/1491b302681a97a3f3c4d73d08eb0197)
<img width="1990" alt="Screenshot 2022-12-28 at 20 32 45" src="https://user-images.githubusercontent.com/1307140/209858978-d2e820d8-8fbc-4294-a7a7-25089015bd69.png">

Trying to install https://github.com/bohdaq/homebrew-rust-tls-server

<img width="2537" alt="Screenshot 2022-12-28 at 20 53 42" src="https://user-images.githubusercontent.com/1307140/209859274-971f0b84-f7a9-4f1c-8156-2ca76a4684ce.png">

**Comment by bohdaq** [comment]
However, installing without --HEAD parameter works fine.

**Comment by cho-m** [comment, marked as answer]
You need to use the repository's branch name, i.e. if https://github.com/bohdaq/rust-tls-server/ uses `main` then the branch should be set to `main`.
```rb
head "https://github.com/bohdaq/rust-tls-server.git", branch: "main"
```

**Comment by bohdaq** [comment]
@cho-m 
Worked perfectly!
